build.sh

```diff
@@ -72,19 +72,25 @@ if [ ! -f "${BUILD_ENV}" ]; then
 fi
 source ${BUILD_ENV}
 
-# Clear out old builds
-if [ -d "${GEOPM_INSTALL}" ]; then
+# Check that GEOPM_INSTALL is defined unless we are skipping install
+if [ -z "${GEOPM_INSTALL}" ] && [ -z "${GEOPM_SKIP_INSTALL+x}" ];
+    echo "Please define GEOPM_INSTALL prior to using this script" 1>&2
+    exit 1
+fi
```

Comment:
If they're using the recommendations from the docs, then they're using ~/.geopmrc and GEOPM_INSTALL is set.  Even if we aren't installing, having GEOPM_INSTALL set is benign.  Why bother?

Additionally, I don't think this is even reachable if GEOPM_INSTALL is not set.  build_env.sh was sourced on line 73, and enforces that GEOPM_INSTALL is set:
https://github.com/geopm/geopm/blob/dev/integration/config/build_env.sh#L49